src/check/model/commands/CommandsArbitrary.ts

```diff
@@ -149,54 +149,12 @@ class CommandsArbitrary<Model extends object, Real, RunResult, CheckAsync extend
  * It should shrink more efficiently than {@link array} for {@link AsyncCommand} arrays.
  *
  * @param commandArbs - Arbitraries responsible to build commands
- * @param maxCommands - Maximal number of commands to build
- *
- * @deprecated
- * Superceded by `fc.commands(commandArbs, {maxCommands})` - see {@link https://github.com/dubzzz/fast-check/issues/992 | #992}.
- * Ease the migration with {@link https://github.com/dubzzz/fast-check/tree/main/codemods/unify-signatures | our codemod script}.
- *
- * @remarks Since 1.5.0
- * @public
- */
-// eslint-disable-next-line @typescript-eslint/ban-types
-function commands<Model extends object, Real, CheckAsync extends boolean>(
```

Comment:
A fixh as to be done on main as `fc.commands([...])` is marked as deprecated while it is not